src/engine/source/kvdb/include/kvdb/kvdbExcept.hpp

```diff
@@ -0,0 +1,80 @@
+#ifndef _KVDB_EXCEPT_H
+#define _KVDB_EXCEPT_H
+
+#include <exception>
+#include <string>
+
+namespace kvdbManager
+{
+
+/**
+ * @brief Exception thrown when critical KVDB Functions Fails
+ */
+class KVDBException : public std::exception
```

Comment:
Is this exception thrown? Also we try to use only standard exceptions

Reply:
Custom exceptions were already removed. Nice catch. No need to use them here at the moment and will rely on std ones. Thank you.